Sources/Defaults/Defaults+iCloud.swift

```diff
@@ -82,15 +82,8 @@ extension Defaults {
 		/**
 		Add the keys to be automatically synced.
 		*/
-		public static func add(_ keys: Defaults.Keys...) {
-			synchronizer.add(keys)
-		}
-
-		/**
-		Add the keys to be automatically synced.
-		*/
-		public static func add(_ keys: [Defaults.Keys]) {
-			synchronizer.add(keys)
+		public static func add<each Value: Defaults.Serializable>(_ keys: repeat Defaults.Key<each Value>) {
+			repeat synchronizer.add(each keys)
 		}
```

Comment:
Untested and would require Swift 6, but maybe:

```swift
public static func add<each Value: Defaults.Serializable>(_ keys: repeat [Defaults.Key<each Value>]) {
	for key in repeat each keys {
		add(key)
	}
}
```

Reply:
Yeah, will add a `TODO` for it first!